src/main/java/org/apache/sling/jcr/repoinit/impl/RepositoryInitializerFactory.java

```diff
@@ -177,5 +192,10 @@ protected void applyOperations(Session session, List<Operation> ops, String logM
         }
     }
 
+    
+    protected boolean continueOnError() {
+        String failOnError = System.getProperty(PROPERTY_FAIL_ON_ERROR,"true");
```

Comment:
Let us please use the OSGi way in order to allow using framework properties as well: https://docs.osgi.org/javadoc/r4v43/core/org/osgi/framework/BundleContext.html#getProperty(java.lang.String)